### PR TITLE
Update to english version and new year

### PR DESCRIPTION
**Pull request by ElBe-Plaq**
Canges made:

- [x] Bot template author name changed
- [x] Year changed (Could be automated)
- [x] Links changed

### CODE REVIEW

**Comment by ElBe-Plaq** [comment]
More changes can be expected.

**Comment by ElBe-Plaq** [comment]
Is `self._last_member = None` needed?

File: Bot.py [hunk 1]
```diff
@@ -19,10 +18,10 @@ async def on_ready():
         try:
             print(f"Loading cog {cog}")
             client.load_extension(cog)
-            print(f"Loaded cog {cog}")
         except Exception as e:
             exc = "{}: {}".format(type(e).__name__, e)
             print("Failed to load cog {}\n{}".format(cog, exc))
-
+        else:
```
Comment (egeyardimci) [inline Bot.py:24]: Not necessary.

File: Functions/Fun/gameinfos.py [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
         minecraft_table = discord.Embed(
             title="BotName",
             colour=discord.Colour.gold()
         )
         
-        minecraft_table.add_field(name="Skull Code Minecraft 1.13 And Higher",value='/give @p minecraft:player_head{p1}SkullOwner:"{name}"{p2}'.format(p1="{",name=name,p2="}"),inline=False)
-        minecraft_table.add_field(name="Skull Code Minecraft 1.13 And Lower",value='/give @p minecraft:skull 1 3 {p1}SkullOwner:"{name}"{p2}'.format(p1="{",name=name,p2="}"),inline=False)
+        p1, p2 = '{', '}'
+        minecraft_table.add_field(name="Skull Code Minecraft 1.13 And Higher",value=f'/give @p minecraft:player_head{p1}SkullOwner:"{name}"{p2}',inline=False)
+        minecraft_table.add_field(name="Skull Code Minecraft 1.13 And Lower",value=f'/give @p minecraft:skull 1 3 {p1}SkullOwner:"{name}"{p2}',inline=False)
```
Comment (egeyardimci) [inline Functions/Fun/gameinfos.py:20]: I would rather leave it as it is.

**Review by egeyardimci — COMMENTED** [review]
A pull request that only corrects the links and automates the data of current year can be approved.

File: setup.bat [hunk 3]
```diff
@@ -1,7 +1,4 @@
+@echo off
 pip install -r requirements.txt
 
-cls
-@echo off
 echo Setup succeed.
-
-PAUSE
```
[the remark this answers is not in the document]
Comment (ElBe-Plaq) [reply setup.bat:7]: PAUSE is not needed, so I removed it.

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,2 @@
-discord
 discord.py
-beautifulsoup4
-urllib3
-requests
```
[the remark this answers is not in the document]
Comment (ElBe-Plaq) [reply requirements.txt:5]: The librarys aren't used.

File: Functions/Info/info.py [hunk 5]
```diff
@@ -30,7 +30,7 @@ async def help(self, ctx):
             colour=discord.Colour.blue()
         )
         info_board.set_footer(text="BotName")
-        info_board.set_author(name="xegepa")
+        info_board.set_author(name="egeyardimci")
```
[the remark this answers is not in the document]
Comment (ElBe-Plaq) [reply Functions/Info/info.py:33]: Okay, I'll change it back.

**Comment by ElBe-Plaq** [comment]
Can it be merged now? @egeyardimci

**Comment by ElBe-Plaq** [comment]
@egeyardimci ?

**Comment by egeyardimci** [comment]
Sorry for delay i was not able to maintain this repo for a while.

**Pull request merged**